modules/local/localcolabfold.nf

```diff
@@ -10,26 +10,24 @@ process RUN_COLABFOLD {
     tuple val(seq_name), path(fasta)
     val model_type
     path db
+    val numRec
```

Comment:
This module needs that the container is updated to version `v0.5`, doesn't it?

Reply:
No, the functional version is still v0.4.